server/pages/index.js

```diff
@@ -359,7 +361,13 @@ module.exports = function() {
 			// the user is probably logged in
 			renderLoggedInRoute( req, res );
 		} else {
-			renderLoggedOutRoute( req, res );
+			const LayoutLoggedOutDesign = require( 'layout/logged-out-design' ),
+				LayoutLoggedOutDesignFactory = React.createFactory( LayoutLoggedOutDesign ),
+				context = getDefaultContext( req );
+
+			context.layout = ReactDomServer.renderToString( LayoutLoggedOutDesignFactory() );
```

Comment:
We should try to catch any `renderToString` errors here, and throw them in `development`, but ignore them in other envs. I think we'd want to only set `context.layout` if there's no exception, so if there is, things _should_ just fall back to client side rendering nicely.